conftest.py

```diff
@@ -0,0 +1,24 @@
+import ee
+import json
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--config", action="store", default="/home/circleci/project/SERVICE_ACCT.json"
+    )
+
+
+@pytest.fixture(scope="session")
+def authenticate_gee(pytestconfig):
+    try:
+        config_path = pytestconfig.getoption("config")
+        with open(config_path) as f:
+            config = json.load(f)
+    except FileNotFoundError as e:
+        raise ValueError(
+            f"Service account config, {config_path}, not found. Please ensure the "
+            "path provided to --config command line argument is correct."
+        )
+    service_acct = config["client_email"]
+    ee.Initialize(ee.ServiceAccountCredentials(service_acct, config_path))
```

Comment:
Where is ee.ServiceAccountCredentials documented? that returns the `baseurl` for ee.Initialize?
If it fails and error is thrown? otherwise success?... i guess I expect an assertion.

Reply:
The only EE docs I can find on it are [these](https://developers.google.com/earth-engine/guides/service_account#use-the-service-account), which is part of why I setup the Circle config to run the API connectivity check first before running the rest of the test suite, in case it failed in awkward ways that created hard to diagnose failures which were actually just due to authentication issues.

Reply:
I'm still learning to navigate the docs! it does have a search feature... 🤦